bvm/ballerina-runtime/src/test/java/io/ballerina/runtime/test/config/negative/CLIProviderNegativeTest.java

```diff
@@ -36,7 +36,7 @@
 /**
  * Test cases specific for configuration provided via cli.
  */
-public class CLIConfigProviderNegativeTest {
+public class CLIProviderNegativeTest {
```

Comment:
Shall we rename to 
```suggestion
public class CliProviderNegativeTest {
```
as it is the normal convention decided upon if I am not mistaken

Reply:
There was another test. Let's check other places and rename accordingly.